src/main/java/com/shopping_app/shoppingApp/model/Product/ProductResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.shopping_app.shoppingApp.model.Product;
+
+import com.shopping_app.shoppingApp.domain.Product;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class ProductResponse {
+
+    private long id;
+
+    private String productName;
+
+    private double productPrice;
+
+    private String productDescription;
+
+    private int availableQuantity;
+
+    public static ProductResponse from(Product product) {
+        if (product == null) {
+            return null;
+        }
```

Comment:
Do we need this?

Reply:
Yes this is not neccessary , while i copying the code from already implemented branch i missed out this to remove .

Reply:
updated